ihatemoney/templates/settle_bills.html

```diff
@@ -25,7 +25,7 @@
     <tr receiver={{bill.receiver.id}}>
             <td>{{ bill.ower }}</td>
             <td>{{ bill.receiver }}</td>
-            <td>{{ "%0.2f"|format(bill.amount) }}</td>
+            <td>{{ bill.amount|currencyformat_nc(g.project.default_currency) }}</td>
```

Comment:
What do you think about creating a shortcut filter so we could do `bill.amount | currency` for instance?

Reply:
I'll try to have a look at it.

The currency filter should then be able to format without parameter (and use project's default), or accept a currency as parameter (for bill list).